Modules/Package.swift

```diff
@@ -48,7 +48,7 @@ let package = Package(
         .package(url: "https://github.com/wordpress-mobile/MediaEditor-iOS", branch: "task/spm-support"),
         .package(url: "https://github.com/wordpress-mobile/NSObject-SafeExpectations", from: "0.0.6"),
         .package(url: "https://github.com/wordpress-mobile/NSURL-IDN", branch: "trunk"),
-        .package(url: "https://github.com/wordpress-mobile/WordPressKit-iOS", branch: "wpios-edition"),
+        .package(url: "https://github.com/wordpress-mobile/WordPressKit-iOS", branch: "task/new-subscibers-api"),
```

Comment:
I have approvied the WPKit PR. This change can be discarded once the WPKit PR is merged.